src/libs/actions/Report.js

```diff
@@ -287,7 +288,15 @@ function subscribeToReportCommentEvents() {
     }
 
     const pusherChannelName = `private-user-accountID-${currentUserAccountID}`;
-    if (Pusher.isSubscribed(pusherChannelName) || Pusher.isAlreadySubscribing(pusherChannelName)) {
+
+    const pusherSubscribedOrSubscribing = Pusher.isSubscribed(pusherChannelName)
+        || Pusher.isAlreadySubscribing(pusherChannelName);
+
+    // This check is to prevent duplicate subscriptions.
+    // Note: When running automated tests the act of calling Pusher.isSubscribed will create a mock
+    // channel already in a subscribed state. So, we must skip this check when tests are running or else it
+    // will prevent us from binding an event to the channel below.
+    if (!CONFIG.IS_JEST_RUNNING && pusherSubscribedOrSubscribing) {
```

Comment:
Is there any way we could mock `subscribeToReportCommentEvents` so we don't have to have this check?

Reply:
So, the way I am testing this now is a sort of verification that the `subscribeToReportCommentEvents()` action is doing what it's supposed to do. Which is, subscribe to the channel and handle the payload.

One think we could do is add a `force` parameter to the action that skips this check and always tries to subscribe to Pusher no matter what. Then we could lose this config. 

But I'd be curious to hear if you have any other ideas on how to handle this!

Reply:
also very curious to learn more about why if it's the case that you are not loving having this be a `CONFIG`.

Reply:
Wait! I've got an idea. We can mock those Pusher methods so they just return false no matter what.